src/main/java/com/revature/controllers/QuestionController.java

```diff
@@ -40,4 +47,56 @@
 		return ResponseEntity.ok(questionLogic.searchQuestions(pageable,searchText, subject));
 	}
 	
+//QUESTIONPOOL CUD------------------------------------
+	@RequestMapping(method = RequestMethod.POST, value="")
+	public ResponseEntity<QuestionPool> addQuestion(@RequestBody QuestionPool question){
+		
+		System.out.println(question);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)